toonz/sources/toonzqt/dvdialog.cpp

```diff
@@ -757,6 +758,29 @@ void Dialog::addButtonBarWidget(QWidget *first, QWidget *second, QWidget *third,
 
 //=============================================================================
 
+MessageAndCheckboxDialog::MessageAndCheckboxDialog(QWidget *parent,
+                                                   bool hasButton,
+                                                   bool hasFixedSize,
+                                                   const QString &name)
+    : Dialog(parent, hasButton, hasFixedSize, name) {}
+
+//=============================================================================
+
+void MessageAndCheckboxDialog::onButtonPressed(int id) {
+  if (m_checked > 0) {
+    done(id + 5);
```

Comment:
You are returning the state of checkbox by adding 5 to the return value.
It's not a problem for the moment, but if the dialog is created with more than 5 buttons, return value will be overlapped. 
How about replacing this magic number with a larger & named constant (say, `const static int  MessageAndCheckboxDialog::CHECKED = 1000` ) in order to surely prevent the overlap? Or, how about adding a separate function which will return `m_checked` value?